prombench/README.md

```diff
@@ -118,3 +118,19 @@ export PR_NUMBER=<PR to benchmark against the selected $RELEASE>
     -v GITHUB_ORG:${GITHUB_ORG} -v GITHUB_REPO:${GITHUB_REPO} \
     -f manifests/prombench/benchmark
 ```
+### Delete and Cleanup
+  - Delete the prombench GKE cluster
+  ```
+  ./prombench gke cluster delete  -a $AUTH_FILE 
+    -v PROJECT_ID:$PROJECT_ID -v ZONE:$ZONE \
+    -v CLUSTER_NAME:$CLUSTER_NAME \ 
+    -f manifests/cluster.yaml
+  ```
+
+  - Delete the GKE node pool
```

Comment:
Deleting the nodepool does not clear the namespace created for the benchmarking test,
`make clean` already addresses this issue. It was decided earlier to have the README show the explicit commands rather than `make deploy`.